### PR TITLE
Update k8 v.19

### PR DESCRIPTION
**Pull request by npolshakova**
# Description

Update Gloo to works with k8 1.19.

# Context

https://github.com/solo-io/gloo/issues/3582

Dependency updates:

- skv2 (v0.16.1 => v0.7.0)
- helm (v3.2.5 => v3.4.2)
- kube-openapi (v0.0.0-20190816220812-743ec37842bf => v0.0.0-20200805222855-6aeccd4b50c6)
- containerd (v1.3.3 => v1.4.3)

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/3582

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -89,29 +87,27 @@ require (
...
 	google.golang.org/grpc v1.33.1
 	google.golang.org/protobuf v1.25.0
 	gopkg.in/AlecAivazis/survey.v1 v1.8.7
 	gopkg.in/ini.v1 v1.56.0 // indirect
 	gopkg.in/src-d/go-git.v4 v4.13.1 // indirect
-	helm.sh/helm/v3 v3.2.4
-	k8s.io/api v0.18.8
-	k8s.io/apiextensions-apiserver v0.18.8
-	k8s.io/apimachinery v0.18.8
+	helm.sh/helm/v3 v3.4.2
```
Comment (npolshakova) [inline go.mod:97]: Need to upgrade helm v3.2.5 -> v3.4.2, as v3.4.x supports k8 1.19 (https://helm.sh/docs/topics/version_skew/#supported-version-skew)

File: test/mocks/kubernetes/kubeinterface.go [hunk 2]
```diff
@@ -144,20 +145,6 @@ func (mr *MockInterfaceMockRecorder) AppsV1beta2() *gomock.Call {
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "AppsV1beta2", reflect.TypeOf((*MockInterface)(nil).AppsV1beta2))
 }
 
-// AuditregistrationV1alpha1 mocks base method
```
Comment (npolshakova) [inline test/mocks/kubernetes/kubeinterface.go:147]: auditregistration.k8s.io/v1alpha1 API have been removed and are no longer supported in 1.19 (https://github.com/kubernetes/kubernetes/pull/91502/files)

**Review by mlholland — APPROVED** [review]

File: go.mod [hunk 3]
```diff
@@ -122,7 +118,10 @@ replace (
 	// Note the correct  version when updating.
 	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.4.2
 	github.com/census-instrumentation/opencensus-proto => github.com/census-instrumentation/opencensus-proto v0.2.0 // indirect
+	// required for ci
```
Comment (kdorosh) [inline go.mod:121]: link to an issue? why is this required?

File: changelog/v1.7.0-beta9/update-k8s-19.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+changelog:
```
Comment (Sodman) [inline changelog/v1.7.0-beta9/update-k8s-19.yaml:1]: Can we also add a `DEPENDENCY_BUMP` type in here? I think doing it for _every_ k8s dependency we bumped in `go.mod` might be overkill, but maybe just for the main `k8s.io/kubernetes v1.19.6` dependency would be helpful for folks looking over the changelog.

File: go.mod (hunk 3 above)
Comment (npolshakova) [reply go.mod:121]: I can change the comment, but this was the error I was getting without bumping the comtainerd version: 

```
Error: ../../../go/pkg/mod/github.com/containerd/containerd@v1.3.4/sys/proc.go:33:34: undefined: system.GetClockTicks
make: *** [/home/runner/work/gloo/gloo/_output/glooctl-linux-amd64] Error 2
Error: Makefile:213: recipe for target '/home/runner/work/gloo/gloo/_output/glooctl-linux-amd64' failed
>> Removing /tmp/tmp.hDUkz37XSV
Error: Process completed with exit code 2.
```

You don't see the issue testing locally, but in the CI environment `system.GetClockTicks` has been replaced in the system dependency so I needed to bump the `containerd` dependency to fix it.
